asserts/store_asserts.go

```diff
@@ -0,0 +1,94 @@
+package asserts
+
+import (
+	"fmt"
+	"net/url"
+)
+
+// EnterpriseStore holds an enterprise-store assertion, defining the
+// configuration needed to connect a device to the enterprise store.
+type EnterpriseStore struct {
```

Comment:
It's now CustomStore/custom-store. Updated all related var names I could find too.